install/migrations/update_10.0.15_to_10.0.16/config.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2024 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+/**
+ * @var \DBmysql $DB
+ * @var \Migration $migration
+ */
+global $DB, $migration;
+
+// Attach existing mail configuration settings log to NotificationMailingSetting object
+$or_criteria = [];
+foreach (NotificationMailingSetting::getRelatedConfigKeys() as $field) {
+    // `old_value` starts with the field name, followed by a space, then the field value
+    $or_criteria[] = [
+        'old_value' => ['LIKE', $field . '\\ %']
+    ];
+}
+$migration->addPostQuery(
+    $DB->buildUpdate(
+        'glpi_logs',
+        [
+            'itemtype' => 'NotificationMailingSetting',
+            'id_search_option' => 1,
+        ],
+        [
+            'NOT' => ['itemtype' => 'NotificationMailingSetting'], // prevent migration to be done twice
```

Comment:
```suggestion
            'itemtype' => 'Config',
```

Right now we are updating logs for potentially every itemtype (except "NotificationMailingSetting").
Since the value we want to migrate are targeting the "Config" itemtype, it would be safer to target it explicitly.

Reply:
Indeed, it is simpier.